### ISSUE TITLE
Add a helper for `Config/Needs/*` field ? 

### ISSUE THREAD

**Issue by cderv** [r-lib/usethis#1544]
Same as we have `usethis::use_package()` to modify the description programmatically, it would be interesting to have a way to configure the `Config/Needs/website` or other key.

This would allow some custom `use_*` helpers to add packages in DESCRIPTION. 

Motivating use case is the rework of the `quillt::use_quillt_pkgdown()` function now that we use tidytemplates. quillt helper is used to setup a pkgdown website in the R Markdown ecosystem. 

Currently, it seems I need to use an internal function 
````r
usethis:::use_description_list("Config/Needs/website", c("tidyverse/tidytemplate", "rstudio/quillt"))
````

**Comment by jennybc** [comment]
I suspect that someone who needs this should just be using the desc package directly. It's not a common or tricky enough operation to justify a new usethis function.

**Comment by cderv** [comment]
Ok I understand. 

It felt like a good complement to other functions that modified imports and suggests though. I found ˋuse_description_list` useful as a general way to create new project workflows that needs to adapt DESCRIPTION file, same way it is used for internal workflow function. 

But I can definitely use an internal function or use desc directly as you suggested. I am definitely not blocked by that as I already worked around it. I was just suggesting. Thanks for having considered it.
